.travis.yml

```diff
@@ -1,6 +1,10 @@
+sudo: false
 language: node_js
 node_js:
   - "6"
-  - "7"
   - "8"
```

Comment:
If you're going to update the list of supported versions, please add 9

Reply:
Nah, I just wanted to remove the non-LTS version.